src/core/components/forms/Select.tsx

```diff
@@ -18,6 +18,7 @@ export type SelectProps<O> = {
   filterOptions?: (options: O[], query: string) => O[];
   displayValue?: (option: O) => string;
   multiple?: boolean;
+  label?: string;
```

Comment:
`label` is too generic. `addLabel` is way better. You can also define a default = "Add"